src/components/Connections/CreateConnectionForm.tsx

```diff
@@ -459,6 +459,7 @@ const CreateConnectionForm = ({
                         <Switch
                           data-testid={`incremental-all-streams`}
                           checked={incrementalAllStreams}
+                          disabled={sourceStreams[0]?.cursorField ? false: true}
```

Comment:
not understanding this check... why is `sourceStreams[0]` special?

Reply:
-So , the `incremental-**all**-streams` buttons will be disabled if there are **no** cursor fields.

Reply:
what i mean is, what if `sourceStreams[0]?.cursorField` is present but `sourceStreams[1]?.cursorField` is not? don't we have to check all the `sourceStreams`

also i don't know if we even want this... should we "disable-all" if even one can't be enabled or if all can't be enabled... i don't know. i just want the logic enforced on a per-stream basis

Reply:
- I have added a code to check `isAnyCursorAbsent` .
- Packed inside useMemo with filterSourceStreams as dependency for efficiency.